benchmark/features/benchmark_assert_type_throw.ts

```diff
@@ -37,15 +37,15 @@ import { IoTsObjectUnionExplicit } from "../structures/io-ts/IoTsObjectUnionExpl
 import { IoTsObjectUnionImplicit } from "../structures/io-ts/IoTsObjectUnionImplicit";
 import { IoTsUltimateUnion } from "../structures/io-ts/IoTsUltimateUnion";
 import { IoTsUtils } from "../structures/io-ts/IoTsUtils";
-// TYPEBOX TYPES
-import { __TypeBoxArrayRecursive } from "../structures/typebox/TypeBoxArrayRecursive";
-import { __TypeBoxArrayRecursiveUnionExplicit } from "../structures/typebox/TypeBoxArrayRecursiveUnionExplicit";
-import { __TypeBoxArrayRecursiveUnionImplicit } from "../structures/typebox/TypeBoxArrayRecursiveUnionImplicit";
-import { __TypeBoxObjectHierarchical } from "../structures/typebox/TypeBoxObjectHierarchical";
-import { __TypeBoxObjectSimple } from "../structures/typebox/TypeBoxObjectSimple";
-import { __TypeBoxObjectUnionExplicit } from "../structures/typebox/TypeBoxObjectUnionExplicit";
-import { __TypeBoxObjectUnionImplicit } from "../structures/typebox/TypeBoxObjectUnionImplicit";
-import { __TypeBoxUltimateUnion } from "../structures/typebox/TypeBoxUltimateUnion";
+// TYPEBOX TYPES (Equals?)
```

Comment:
Using `Equals` schemas here does not look correct to me, but note these do allow these tests to pass and I don't know why. This could suggest the `assertType()` is internally using the `Equals` checks (not the `Is` checks) but this doesn't seem likely to me. Will leave as is for review.